File: java_db_service/src/main/java/ru/mipt/osint_db_service/repository/webParsing/WebSourcesLinkRepository.java
package ru.mipt.osint_db_service.repository.webParsing;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import ru.mipt.osint_db_service.model.webParsing.WebSourcesLink;

@Repository
public interface WebSourcesLinkRepository extends JpaRepository<WebSourcesLink, Long> {
}

File: Osint_web_service/src/main/kotlin/ru/bitok/osint/osint_web_service/repository/WebSourcesLinkRepository.kt
package ru.bitok.osint.osint_web_service.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import ru.bitok.osint.osint_web_service.entity.WebSourcesLink

@Repository
interface WebSourcesLinkRepository: JpaRepository<WebSourcesLink, String> {
    fun findByLink(link:String):WebSourcesLink?
}